lib/index.js

```diff
@@ -27,11 +27,22 @@ module.exports = (expectedScopes, options) => {
     if (expectedScopes.length === 0) {
       return next();
     }
-    if (!req.user || typeof req.user.scope !== 'string') {
+
+    let userScopes = [];
+
+    if (!req.user) {
+      return error(res);
+    }
+
+    if (typeof req.user.scope === 'string') {
+      userScopes = req.user.scope.split(' ');
+    } else if (Array.isArray(req.user.scope)) {
+      userScopes = req.user.scope;
+    } else {
       return error(res);
     }
-    const scopes = req.user.scope.split(' ');
-    const allowed = expectedScopes.some(scope => scopes.includes(scope));
+
+    const allowed = expectedScopes.every(scope => userScopes.includes(scope));
```

Comment:
Here's the breaking change. Again, if this is a problem, I can make it optional.